technologies/technologies.json

```
@@ -11,6 +11,12 @@
         "displayName": "Laravel",
         "description": "Laravel is a free, open-source PHP web framework, created by Taylor Otwell and intended for the development of web applications following the model–view–controller (MVC) architectural pattern and based on Symfony."
     },
+    {
+        "id": "javascript",
+        "isVisible": true,
```

If you don't mind I think it is better to keep `isVisible` as `false` until we have the list almost ready.

```suggestion
        "isVisible": false,
```

---

I set the isVisible flag to false